litrpc/lit-accounts.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+
+package litrpc;
+
+option go_package = "github.com/lightninglabs/lightning-terminal/litrpc";
+
+service Accounts {
+    /*
+    CreateAccount adds an entry to the account database. This entry represents
+    an amount of satoshis (account balance) that can be spent using off-chain
+    transactions (e.g. paying invoices).
+
+    Macaroons can be created to be locked to an account. This makes sure that
+    the bearer of the macaroon can only spend at most that amount of satoshis
+    through the daemon that has issued the macaroon.
+
+    Accounts only assert a maximum amount spendable. Having a certain account
+    balance does not guarantee that the node has the channel liquidity to
+    actually spend that amount.
+    */
+    rpc CreateAccount (CreateAccountRequest) returns (CreateAccountResponse);
+
+    /*
+    ListAccounts returns all accounts that are currently stored in the account
+    database.
+    */
+    rpc ListAccounts (ListAccountsRequest) returns (ListAccountsResponse);
+
+    /*
+    RemoveAccount removes the given account from the account database.
+    */
+    rpc RemoveAccount (RemoveAccountRequest) returns (RemoveAccountResponse);
+}
+
+message CreateAccountRequest {
+    /*
+    The initial account balance in satoshis representing the maximum amount that
+    can be spent.
+    */
+    uint64 account_balance = 1;
+
+    /*
+    The expiration date of the account as a timestamp. Set to zero to never
+    expire.
+    */
+    int64 expiration_date = 2;
+}
+
+message Account {
+    /*
```

Comment:
Should we also return the raw macaroon here? Otherwise how will it be obtained? 

One architecture I have in mind is:
  * Service maintains an external DB of the macaroon for each user's accoutn 
  * Each time a user wants to do something, the services get the macaroon then issues the response as normal

So the service just needs to handle the mapping from user to account macaroon. 

One other alternative  architecture would be the discard model: the account service actually encodes the latest balance in the _macaroon itself_. This minimizes the state, as its the clients jobs to maintain this macaroon, and they can't modify it w/ breaking the HAMC check.

Reply:
Added the baking of an account macaroon to the RPC server and updated the docs.